sys/arm64/arm64/machdep.c

```diff
@@ -228,22 +228,14 @@ int
 fill_regs(struct thread *td, struct reg *regs)
 {
 	struct trapframe *frame;
-#if __has_feature(capabilities)
-	int i;
-#endif
 
 	frame = td->td_frame;
 	regs->sp = frame->tf_sp;
 	regs->lr = frame->tf_lr;
 	regs->elr = frame->tf_elr;
 	regs->spsr = frame->tf_spsr;
 
-#if __has_feature(capabilities)
```

Comment:
This isn't a no-op change.  regs->x is an array of uint64_t.  frame->tf_x is an array of uintcap_t.  I think this diff needs to stay (and be merged up to dev even which I had on my todo list to merge)

Reply:
yeah, I've dropped this change.